README.md

```diff
@@ -92,7 +92,16 @@ resource in an EVRYTHNG account.
 
 In a nutshell, `evrythng.js` provides the following scopes. Once a scope is
 created it provides an appropriate API for the resources it can manage
-(see [API](#api) below):
+(see [API](#api) below).
+
+For `apiVersion:2` next scopes are avaliable:
+
+- `Operator`
+- `AccessToken`
+
+`Operator` and `AccessToken` scopes can have a different set of permissions, which is defined on [access policy](https://developers.evrythng.com/reference/access-policies-1) and assigned during creation of [operator access](https://developers.evrythng.com/reference/operator-access) and [access token](https://developers.evrythng.com/reference/access-tokens).
```

Comment:
I fixed the slug for that page just now:
`https://developers.evrythng.com/reference/access-policies`

Reply:
fixed, thanks